lib/tools/emacs/erlang.el

```diff
@@ -1933,8 +1947,86 @@ The format is described in the documentation of `erlang-man-dirs'."
       '(("Man Pages"
          (("Error! Why?" erlang-man-describe-error)))))))
 
-(defun erlang-man-dir (subdir)
-  (concat erlang-root-dir "/lib/erlang/" subdir))
+
+(defcustom erlang-man-download-url "http://erlang.org/download/otp_doc_man_22.1.tar.gz"
```

Comment:
It would be neat with some automatic update of this.  Perhaps a test in emacs_SUITE that compares this to the current OTP version.

It would also be neat if it refreshed the downloaded man page cache whenever erlang-man-download-url changed.  Maybe the value of erlang-man-download-url could be saved in a file along with the unpacked tarball. And then the content of this file is compared to erlang-man-download-url every time.  If it differ delete the cache and download the new url.

Reply:
Or for now at least just put the value of erlang-man-download-url in a file so that a future version of erlang.el would know if it would be appropriate to replace it.

Reply:
I have changed erlang-man-download so that it will update the man cache if the URL has changed since the man pages got downloaded in commit 8157710. I agree that it would be neat to automatically update the default value for erlang-man-download-url when a new Erlang version is released but I will leave this for the future. Thanks for the suggestion!

Reply:
Thanks, looks good to me.